panther_analysis_tool/main.py

```diff
@@ -903,39 +907,27 @@ def setup_run_tests(  # pylint: disable=too-many-locals,too-many-arguments
         if skip_disabled_tests and not analysis_spec.get("Enabled", False):
             continue
         analysis_type = analysis_spec["AnalysisType"]
-        analysis_id = analysis_spec.get("PolicyID") or analysis_spec["RuleID"]
+
+        detection_args = dict(
+            id=analysis_spec.get("PolicyID") or analysis_spec["RuleID"],
+            analysisType=analysis_type.upper(),
```

Comment:
The analysis type needs to be upper case to work in panther core. I am not sure how this was working before without that